demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -122,6 +127,28 @@ def _quit_checker(self, node):
         except Exception:
             pass
 
+    def _execute_command_checker(self, node):
+        if not self.is_script or self.runas == 'DBotRole':
+            return
+
+        try:
+            if node.func.expr.name == 'demisto' and node.func.attrname == 'executeCommand':
+
+                if node.args[0].value == 'getIncidents' or node.args[0].value == 'DeleteContext':
```

Comment:
```suggestion
                if node.args[0].value in ['getIncidents', 'DeleteContext']:
```

Reply:
done.
Also added here the additional command, and put them in the constants file.